climada/hazard/trop_cyclone.py

```diff
@@ -485,25 +485,17 @@ def _apply_knutson_criterion(self, chg_int_freq, scaling_rcp_year):
                 sel_cat_chg = np.isin(tc_cc.category, cat_chg_list) & bas_sel
                 if sel_cat_chg.any():
                     freq_scaling = 1 + (chg['change'] - 1) * scaling_rcp_year
```

Comment:
```suggestion
                    if chg['change'] >= 1:
                          freq_scaling = 1 + (chg['change'] - 1) * scaling_rcp_year
                    else:
                          freq_scaling = chg['change'] / scaling_rcp_year
```

Reply:
With the current implementation, a Knutson factor of 0.9 with a scaling of 3 would become 0.7. With your suggested change, the Knutson factor 0.9 would become 0.3. So, that's not very reasonable, right?

Reply:
You have a good example. To be honest, I would not say that it is not reasonable. It really depends on how one does interpret the extrapolation. 

The extrapolation I suggest is "a doubling in temperature leads to a doubling of the relative frequency/intensity change if is increasing, and to a halving of the relative frequency/intensity change if it is decreasing". This seems reasonable to me, but as you are saying, it is not necessarily  the best interpolation method.

Reply:
I will now merge with the old method and open an issue to discuss this point. I addressed the points which you raised in https://github.com/CLIMADA-project/climada_python/pull/223#discussion_r629454061